components.inky.js

```diff
@@ -0,0 +1,364 @@
+function getElementClasses(element) {
+  var classes = [];
+  if (element.attr('class')) {
+    classes = classes.concat(element.attr('class').split(' '));
+  }
+  return classes.join(' ');
+}
+
+module.exports = {
```

Comment:
Ahh, I see. Should've realised, the file was already getting out of hand with even a handful of components!

No worries though, will re-organise